sim/team-validator.ts

```diff
@@ -832,6 +822,17 @@ export class TeamValidator {
 			problems.push(...this.validateStats(set, species, setSources, pokemonGoProblems));
 		}
 
+		// Attempt move validation again after verifying Pokemon GO origin
+		if (setSources.isFromPokemonGo) {
```

Comment:
I think you can move the obtainablemoves check up here rather than later in the if statement - it makes it clear that this won't run if it's Custom Game etc.

Reply:
Or maybe validateMoves() should just always return early if obtainablemoves isn't in the ruleTable? That would catch this problem from occurring in the future as well.